Testing/SystemTests/tests/analysis/AbinsTest.py

```diff
@@ -451,3 +451,27 @@ def runTest(self):
     def validate(self):
         self.tolerance = 1e-2
         return self._output_name, self.ref_result
+
+
+class AbinsCASTEPIsotopes(stresstesting.MantidStressTest, HelperTestingClass):
+    """
+    In this benchmark it is tested is calculations for systems with isotope substitution works properly. Input data is
```

Comment:
Minor: could you reword "In this benchmark it is tested is calculations for systems with isotope substitution works properly" please